packages/shared/src/hooks/feed/useFeedInfiniteScroll.ts

```diff
@@ -5,7 +5,7 @@ import OnboardingContext, {
 } from '../../contexts/OnboardingContext';
 
 export default function useFeedInfiniteScroll(
-  fetchPage: () => Promise<void>,
+  fetchPage: () => Promise<void | unknown>,
```

Comment:
```suggestion
  fetchPage: () => Promise<unknown>,
```

Reply:
Hi, I actually had to do this as the tslint is acting very strange. It is giving an error to the below declaration:

<img width="411" alt="Screen Shot 2021-09-23 at 11 17 49 PM" src="https://user-images.githubusercontent.com/13744167/134535295-bae1e4d5-e406-4647-83e1-1593d290ec9f.png">

Which if not allowed, the parameter will be verbose just to be accepted:

<img width="410" alt="Screen Shot 2021-09-23 at 11 21 40 PM" src="https://user-images.githubusercontent.com/13744167/134535974-f42c5ed7-d0ad-44d1-a2e2-585a84cfdc5d.png">